dora/core/common/src/main/java/alluxio/membership/EtcdMembershipManager.java

```diff
@@ -204,6 +204,6 @@ public void decommission(WorkerInfo worker) throws IOException {
 
   @Override
   public void close() throws Exception {
-    mAlluxioEtcdClient.close();
+    // NOTHING TO CLOSE
```

Comment:
so will never close the client?

Reply:
So AlluxioEtcdClient will be used as a singleton throughout entire process lifecycle, all reconnect/loadbalance logic is already handled by underlying jetcd client, I don't want to add such logic at our layer to complicate things as of now. And as MembershipManager is also used in client, such as FileSystemContext, which is constantly closed and reopen, it shouldn't be closing AlluxioEtcdClient as its shared with others.